plugin-web/src/main/java/nl/xillio/xill/plugins/web/services/web/WebServiceImpl.java

```diff
@@ -355,7 +361,15 @@ public void quit(final WebVariable var) {
 
     @Override
     public PhantomJSPool.Entity getEntityFromPool(final PhantomJSPool pool, final Options options) {
-        PhantomJSPool.Entity pjsInstance = pool.get(pool.createIdentifier(options), this);
+        PhantomJSPool.Entity pjsInstance = null;
```

Comment:
This is in the Java conventions as an anti-pattern:

https://help.xillio.com/conventions/java-code/#limit-scope-as-much-as-possible

Reply:
I am just curious - is mentioned case considered to be anti-pattern only because someone in Xillio does not like it or is there any solid explanation from some Java expert?

Reply:
I don't exactly know however I think the explanation given is valid:
`The scope of local variables can often be limited to increase readability, testability and general maintainability of the code. `

So I think in general it is good to do it.